srv/modules/runners/populate.py

```diff
@@ -25,6 +25,7 @@
 import ipaddress
 import logging
 import deepsea_minions
+from validate import Validate, get_printer
```

Comment:
This line appears to be triggering the following error, which (apparently) causes Stage 3 to fail when running the qa suite in teuthology:

```
2017-08-25T19:49:32.861 INFO:teuthology.orchestra.run.target217182143124.stdout:Traceback (most recent call last):
2017-08-25T19:49:32.861 INFO:teuthology.orchestra.run.target217182143124.stdout:  File "/usr/lib/python2.7/site-packages/salt/loader.py", line 1333, in _load_module
2017-08-25T19:49:32.862 INFO:teuthology.orchestra.run.target217182143124.stdout:    mod = imp.load_module(mod_namespace, fn_, fpath, desc)
2017-08-25T19:49:32.862 INFO:teuthology.orchestra.run.target217182143124.stdout:  File "/srv/modules/runners/populate.py", line 28, in <module>
2017-08-25T19:49:32.862 INFO:teuthology.orchestra.run.target217182143124.stdout:    from validate import Validate, get_printer
2017-08-25T19:49:32.862 INFO:teuthology.orchestra.run.target217182143124.stdout:ImportError: cannot import name Validate
```